tests/appsec/iast/test_iast.py

```diff
@@ -2,36 +2,64 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import BaseTestCase, interfaces, context, missing_feature, coverage
-
+from utils import BaseTestCase, interfaces, context, missing_feature, coverage, released
 
 # Weblog are ok for nodejs/express4 and java/spring-boot
```

Comment:
if I understodd correctly, the IAST feature is not released in any library (aka tracer), we must keep the `@missing_feature(reason="Need to be implement in iast library")`. missing_feature can be removed once the feature is done AND the weblog expose a way to activate the feature.

Reply:
Ok, I will change it. But If Java changes are in the master branch... Are the system-test launched on last snapshot from master or on last public release?